src/Perf.h

```diff
@@ -0,0 +1,78 @@
+#ifndef PERF_H
+#define PERF_H
+
+#include "AtomicMCDAObject.h"
+#include <iostream>
+#include <string>
+
+class Perf : public AtomicMCDAObject {
+public:
+  /**
+   * Perf standard constructor
+   *
+   * @param name name of the performance (ex: name of profile or alternative)
+   * @param criterion criterion id on which the performance in based on
+   * @param value value of the performance
+   */
+  Perf(std::string name, std::string criterion, float value);
+
+  /**
```

Comment:
Maybe we could add  new constructors for example : 
- build a Perf object with only name
- build a Perf object with only criterion
- build a Perf object with name and criterion
- build a Perf with only its value 
I think the third constructor might come in handy, the others they might but not sure ....

Reply:
I will add the third one with default value of 0 but the others are not likely to be useful in our case: we need to know which criterion is used and from where is the data from (name parameter)